wandb/sdk/artifacts/artifact.py

```diff
@@ -146,7 +146,7 @@ def __init__(
             int, Tuple[data_types.WBValue, ArtifactManifestEntry]
         ] = {}
         self._added_local_paths: Dict[str, ArtifactManifestEntry] = {}
-        self._save_future: Optional[MessageFuture] = None
+        self._save_handle: Optional[MailboxHandle] = None
```

Comment:
`MailboxHandle` is only defined during type checking, it has to be a forward reference:
```suggestion
        self._save_handle: Optional["MailboxHandle"] = None
```

Reply:
done